mac/README.md

```diff
@@ -4,18 +4,27 @@
 These steps are meant to provide transparency instead of running installers which works magically sometimes but more frustrating 
 when it doesn't. Let's begin!
 
+#### Mac OS Catalina (10.15 ) and later Steps
 
-#### Steps
+1. Copy fonts and keyboard to your computer, run the following command in your Terminal app <img src="terminal-icon.png" width="30"/>
 
-1. To copy fonts and keyboards to your computer, run the following command in your Terminal app <img src="terminal-icon.png" width="30"/>
+                curl -s https://raw.githubusercontent.com/imechemi/monlam-bodyig/master/mac/install.sh | zsh

```

Comment:
Doesn't bash work on Catalina and later?

Reply:
If bash still works, it's better to keep it as bash and keep the readme as short as possible.